src/app/shared/pipes/reference-brief.ts

```diff
@@ -53,7 +53,7 @@ export class ReferenceBriefPipe implements PipeTransform {
     }
 
     arxivEprint = item.get('arxiv_eprint', '');
-    reportNumber = item.get('report_number', '');
+    reportNumbers = item.get('report_numbers', '');
```

Comment:
I guess we wanna show only the first one right? `item.getIn(['report_numbers', 0], '')`